server/src/resolvers/student_teacher.ts

```diff
@@ -0,0 +1,52 @@
+import { StudentTeacher } from "../entities/StudentTeacher";
+import { MyContext } from "src/types";
+import { Arg, Ctx, Field, InputType, Mutation, ObjectType, Query, Resolver } from "type-graphql";
+import { EntityManager } from "@mikro-orm/postgresql";
+
+
+@InputType()
+class StudentTeacherInput {
+    @Field()
+    student: string;
+    @Field()
+    teacher: string;
+    
+}
+
+@ObjectType()
+class StudentTeacherResponse {
+    @Field(() => StudentTeacher, { nullable: true })
+    studentTeacher?: StudentTeacher
+}
+
+@Resolver()
+export class StudentTeacherResolver {
+
+    @Query(() => StudentTeacher, { nullable: true })
+    getStudentTeacher(
+        @Arg( 'teacher' ) teacher: string,
+        @Ctx() { em }: MyContext
+    ): Promise<StudentTeacher | null>{
+        return em.findOne(StudentTeacher, { teacher });
```

Comment:
Doesn't this just get one student using the teacher id? We want the entire student list right?

Reply:
ya, sorry, i still trying getting a list of data, yesterday i keep getting null values from database.